server/graphql/v2/query/ExpensesQuery.ts

```diff
@@ -56,10 +56,14 @@ const ExpensesQuery = {
       type: ISODateTime,
       description: 'Only return expenses that were created after this date',
     },
+    searchTerm: {
+      type: GraphQLString,
+      description: 'The term to search',
+    }
   },
   async resolve(_, args, req): Promise<CollectionReturnType> {
     const where = {};
-    const include = [];
+    let include = [];
```

Comment:
Let's maybe keep that `const` and `push` to make sure we won't overwrite the variable in the future

Reply:
Done. :+1: